samples/nrf9160/serial_lte_modem/src/slm_at_host.c

```diff
@@ -62,6 +63,12 @@ static size_t at_buf_len;
 static struct k_work cmd_send_work;
 static const char termination[3] = { '\0', '\r', '\n' };
 
+/* forward declaration */
+int slm_at_host_uninit(void);
+
+/* extern functions */
+extern void enter_sleep(void);
```

Comment:
All function declarations are `extern` by default, adding this keyword is redundant.

Reply:
Thanks, applied.